test/parallel/test-buffer-tostring-ascii.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+// Flags: --expose-gc
+
+const common = require('../common');
+const assert = require('assert');
+
+// Maximum length in bytes that V8 can handle.
+// This equals to 268435440 bytes = (256 * 1024 * 1024) - 16
+const kMaxLength = (1 << 28) - 16;
```

Comment:
ah crap. keep forgetting this is for v0.12. sorry. maybe I should hurry and back port those...